tools/mock-okta/util.js

```diff
@@ -224,6 +229,16 @@ util.mapRequestToCache = (req) => {
     data.cookie = headers.cookie;
   }
 
+  // We are redirecting to the client callback. We need to replace the state query parameter with
+  // the state stored in the tape before the redirect
+  else if (req.url.indexOf('/oauth2/v1/authorize/redirect') == 0) {
```

Comment:
Doesn't this need to include the `/default` authorization server path?

Reply:
I don't think we need to handle that path as there are no specific parameters passed that would need massaging by the mock server. Is there anything specific you're thinking about in the auth server path, that needs to be handled by the mock server?

Reply:
Interesting. It does look like Okta removes the *authServerId* for this call. 👍